csrc/lower_bank_conflict.cpp

```diff
@@ -119,9 +169,12 @@ std::vector<int64_t> evaluateAddressesOnFirstPhase(
       }
     }
     auto index_val = replaceBaseAddrWithZero(ti->index());
-    std::cout << index_val->toInlineString() << std::endl;
     int64_t index = expr_eval.evaluate(index_val)->as<int64_t>();
-    addresses.emplace_back(index * word_size_bytes);
```

Comment:
This is a bug, it shouldn't be `word_size_bytes` but `dataTypeSize(*(ti->getDataType()))`.